lib/FloatingButton/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import commentSVG from './../../assets/icons/comment.svg';
+
+const FloatingButton = (props) => {
+  return (
+    <button
+      style={props.position}
```

Comment:
I don't think the positioning should be handled in `gather-content-ui`. With the current name (FloatingButton), this makes sense but when we resolve the other pieces of feedback then it will become clearer, the position will feel out more of place.